script.js

```diff
@@ -23,3 +23,239 @@ links.forEach((link) => {
     document.body.style.overflow = 'visible';
   });
 });
+/** ***************************** *
+Code for Dinamically displaying the cards.
+********************************** */
+
+const cards = [
+  {
+    class: 'flex',
+    header2: 'Tonic',
+    heading3: 'CANOPY',
+    list1: ['Back End Dev', '2015'],
+    paragraph: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    mobileImage: 'images/card1-image.png',
+    desktopImage: 'images/card1.png',
```

Comment:
- [ ] Great job with the responsiveness. Please, you are supposed to have just one image which works for both the mobile and the desktop version. You just have to make them responsive. Please, kindly use one image per array object. This will help when you are adding images of your projects, as you will save time in adding one instead of two.